sdk/src/sysvar/recent_blockhashes.rs

```diff
@@ -39,8 +39,9 @@ impl<'a> FromIterator<&'a Hash> for RecentBlockhashes {
 }
 
 impl Sysvar for RecentBlockhashes {
-    fn biggest() -> Self {
-        RecentBlockhashes(vec![Hash::default(); MAX_ENTRIES])
+    fn size_of() -> usize {
+        // hard-coded so that we don't have to construct an empty
+        1032 // golden, update if MAX_ENTRIES changes
```

Comment:
How is this calculated and why hard-coded and not calculated?

Reply:
This comment should at least reference the test that checks that this value is still valid and the test contain a comment saying that the test is required to ensure the hard-coded value doesn't rot